### PR TITLE
Renamed client-redesign to client

### PR DESCRIPTION
**Pull request by ArsalaBangash**

### CODE REVIEW

File: deployment/server.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ app.use(history({
 app.use(staticFileMiddleware)
 
 app.get('/', function (req, res) {
-  res.render('../client-redesign/dist/index.html')
+  res.render('../app/dist/index.html')
 })
```
Comment (alir128) [inline deployment/server.js]: This would be client/ not app/ as its done in the docker file.

File: deployment/server.js [hunk 2]
```diff
@@ -131,7 +131,7 @@ app.get("/status", (req, res) => {
   console.log(rooms);
 });
 
-const staticFileMiddleware = express.static('../client-redesign/dist')
+const staticFileMiddleware = express.static('../app/dist')
```
Comment (alir128) [inline deployment/server.js]: This would be client/ not app/.

File: deployment/server.js [hunk 3]
```diff
@@ -131,7 +131,7 @@ app.get("/status", (req, res) => {
...
```
Comment (Lakshya2610) [inline deployment/server.js]: Shouldn't this be client instead of app?

File: deployment/server.js [hunk 4]
```diff
@@ -141,7 +141,7 @@ app.use(history({
 app.use(staticFileMiddleware)
 
 app.get('/', function (req, res) {
-  res.render('../client-redesign/dist/index.html')
+  res.render('../app/dist/index.html')
```
Comment (Lakshya2610) [inline deployment/server.js]: Same here

**Pull request merged**